lib/hotel_manager.rb

```diff
@@ -0,0 +1,90 @@
+require 'date'
+require_relative 'room'
+require_relative 'reservation'
+require_relative 'date_range'
+require_relative 'block'
+
+module HotelManagementSystem
+    class HotelManager
+        attr_reader :reservations
+
+        COST = 200
+
+        def initialize
+            @reservations = []
+            @rooms = []
+            @blocks = []
+
+            1..20.times do |i|
+                room = HotelManagementSystem::Room.new(room: i + 1)
```

Comment:
Ruby will be looking for names locally first, prioritizing local names, since they're all in the same module, there's no need to namespace `Hotel::Room`. `Room` should just work! Same with the rest of the namespacing in the code.